docs/experiment/sdks/jvm-sdk.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Experiment JVM SDK (Beta)
+description: Official documentation for Amplitude Experiment's server-side JVM SDK implementation.
+icon: material/language-java
+---
+
+Official documentation for Amplitude Experiment's server-side JVM SDK implementation. This SDK may be used in **either Java or Kotlin** server-side implementations.
+
+!!!info "SDK Resources"
+    [:material-github: Github](https://github.com/amplitude/experiment-jvm-server) · [:material-code-tags-check: Releases](https://github.com/amplitude/experiment-jvm-server/releases)
+
+This documentation is split into two sections for [remote](../general/evaluation/remote-evaluation.md) and [local](../general/evaluation/local-evaluation.md) evaluation:
+
+* [Remote evaluation](#remote-evaluation)
+* [Local evaluation (alpha)](#local-evaluation-alpha)
```

Comment:
Do we need to add cookie session for remote?

Reply:
Yeah, I'm going to add that after the initial release. Good callout though.